source/webhook-trigger.py

```diff
@@ -12,30 +12,53 @@
 import jinja2
 import sys
 
-MIN_PYTHON = (3, 7)
-
-if sys.version_info < MIN_PYTHON:
-    sys.exit(
-        "Found Python "
-        + sys.version_info
-        + " but Python %s.%s or later is required.\n" % MIN_PYTHON
-    )
-
 
 VARIABLES = {
     "CONCOURSE_URL": "The URL through which we'll force resource checks",
     "SLACK_WEBHOOK_URL": "The URL of the Slack webhook we will use to publish messages",
 }
 
+MIN_PYTHON = (3, 7)
```

Comment:
I'd probably leave this and the version check statement at the top and certainly together

Reply:
It would appear that I am declaring it twice. Typo. I'll remove.

Reply:
The `send_slack_error_message` function needs to be defined for the py version check to use it. I have therefore moved the version check to the bottom outside of all the function definitions.